File: src/main/java/com/tms/csp/util/Count.java
package com.tms.csp.util;

public class Count {

    private int count;

    public Count() {
        this(0);
    }

    public Count(int count) {
        this.count = count;
    }

    public void increment() {
        count++;
    }

    public int getCount() {
        return count;
    }
}

File: src/main/java/com/smartsoft/csp/util/Count.kt
package com.smartsoft.csp.util

class Count @JvmOverloads constructor(count: Int = 0) {

    var count: Int = 0
        private set

    init {
        this.count = count
    }

    fun increment(amount: Int = 1) {
        count += amount
    }
}
